### ISSUE TITLE
duplicated advanced power cell after 100 infused

### ISSUE THREAD

**Issue by spx** [McJty/RFTools#1229]
i just got second 100% infused advanced power cell in my inventory

```
[21:33:24] [Server thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_131]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_131]
	at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
Caused by: java.lang.NullPointerException
	at mcjty.rftools.blocks.powercell.PowerCellBlock.func_180633_a(PowerCellBlock.java:250) ~[PowerCellBlock.class:?]
	at net.minecraft.item.ItemBlock.placeBlockAt(ItemBlock.java:171) ~[acl.class:?]
	at net.minecraft.item.ItemBlock.func_180614_a(ItemBlock.java:54) ~[acl.class:?]
	at mcjty.lib.compat.CompatItemBlock.clOnItemUse(CompatItemBlock.java:48) ~[CompatItemBlock.class:?]
	at mcjty.lib.compat.CompatItemBlock.func_180614_a(CompatItemBlock.java:44) ~[CompatItemBlock.class:?]
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:780) ~[ForgeHooks.class:?]
	at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:145) ~[adz.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:493) ~[lv.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:679) ~[me.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:55) ~[jj.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:11) ~[jj.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[fl$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_131]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_131]
	at net.minecraft.util.Util.func_181617_a(SourceFile:45) ~[h.class:?]
	... 5 more

```

**Comment by McJty** [comment]
More info please? What version of rftools and what did you do exactly?

**Comment by OneHalfNinja** [comment]
This happened to me too just now. I had 3 Advanced power cells in my inventory I just made. I infused the first one and placed it and then I placed the other 2 to compare the differences with infused and not infused and that's when I noticed a 4th 100% infused Advanced cell. What's weird is if I put down just one of the either of Advanced infused cell's then it says 4,000,000 rf but if I put them both down they both then change to 8,000,000 rf each. Maybe this is by design and you just have a lucky chance of it happening "idk" but infusing created a second one and these two are linked and if they are both used then it doubles the rf storage on each of them. This isn't a problem for me since I am just playing SP but figure I would add that it happened to me too. I play MC Eternal through Twitch with Auto Updating turned on. I have the latest version of x64 Java and running 1.12.2. I don't need this fixed for me and this persons ticket can remain closed, just an fyi.
